src/depends/json_spirit/CMakeLists.txt

```diff
@@ -3,8 +3,5 @@ json_spirit_reader.cpp
 json_spirit_value.cpp
 json_spirit_writer.cpp)
 
-FIND_PACKAGE(Boost 1.34 REQUIRED)
```

Comment:
❓ This one says version 1.34, but others said 1.33. I used 1.33 in the updated CMakeLists.txt file. Should it be changed to something else?

Reply:
:memo: I'm trying to figure out how to more easily install specific Boost versions in Travis CI for build validation.